src/app/containers/Include/index.jsx

```diff
@@ -23,14 +23,80 @@ const decodeHTML = (str) => {
   return str.replace(replacementsRegex, (match) => replacedParts[match]);
 };
 
-const IncludeContainer = ({ html, type }) => {
+const IncludeContainer = ({ html, type, href }) => {
   const { enabled } = useToggle('include');
+  // const [data, setData] = useState('');
+  const [includeHtml, setIncludeHtml] = useState('');
+  const [scriptTags, setScriptsTags] = useState([]);
 
   const supportedTypes = {
     idt2: 'idt2',
     vj: 'vj',
   };
 
+  const createAppendScriptTag = (code, src) => {
+    return new Promise((resolve, reject) => {
+      const script = document.createElement('script');
+      if (src) {
+        script.src = src;
+        // eslint-disable-next-line func-names
+        script.onload = function () {
+          resolve();
```

Comment:
How come we need to wait for the script to load here? This didn't seem to be necessary in the POCs?

Reply:
For IDT2, we noticed when adding the scripts one after the other regularly, that a script that made use of `IDT.init(...)` was executed before a script that defined `IDT` was fetched and executed, even though the script that defined `IDT` came first.

See this [fiddle](https://jsfiddle.net/2vx0hgpn/) where I've demonstrated this behaviour. Inspect the console, and note that the error, `IDT is not defined` is thrown.

Reply:
Ah I can see the issue, might be nice to do a call with the rest of the team to take us through the problem?